webapp/platform/components/src/compass-ui/themeprovider/themeprovider.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {createTheme, ThemeProvider as MUIThemeProvider} from '@mui/material/styles';
+
+import {lightTheme} from './themes';
+import overrides from './overrides';
+
+const ThemeProvider = ({theme = lightTheme, ...rest}) => {
```

Comment:
Just a thought, but we could make this take a MM theme and automatically convert it into a MUI one rather than making the person using the ThemeProvider do that conversion itself. That'd be nice since it would further abstract away the MUI parts of this